src/test/fixtures/architectures.ts

```diff
@@ -1,8 +1,25 @@
-export const mockArchitecturesByDistro = (distro) => {
-  if (distro === 'rhel-92') {
-    return [
+import { Architectures, Distributions } from '../../store/imageBuilderApi';
+
+export const mockArchitecturesByDistro = (
+  distro: Distributions
+): Architectures => {
+  const mockDistros = {
```

Comment:
Let's try typing mockDistros and removing the explicit `Architectures` return type from the function, letting TS infer it instead.